src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -13,7 +13,7 @@ internal static class LogMessages
     {
 #pragma warning disable 1591
         // general
-        // public const string IDX10000 = "IDX10000:";
+        public const string IDX10000 = "IDX10000: The parameter '{0}' cannot be a 'null' or an empty object. ";
```

Comment:
We shouldn't reuse a string that got obsoleted. you'd want to create a new constant (incrementing the numbers)

Reply:
Currently there is a method, `LogArgumentNullException`, that has the following code:

`return LogArgumentException<ArgumentNullException>(EventLevel.Error, argument, "IDX10000: The parameter '{0}' cannot be a 'null' or an empty object. ", argument);`

I figured we are using IDX10000 for this very message, so instead of repeating this pattern, I would add it to `LogMessages.cs` and consume the constant.

Reply:
We took the string out and embedded the code in LogHelper, i think it is OK to bring it back.